components/track/TrackItem.vue

```diff
@@ -8,21 +8,60 @@ defineProps<{
 </script>
 
 <template>
-  <li class="py-3 rounded-xl flex gap-3 items-center">
-    <ProtectedImage
-      v-if="track.meta?.attachedPicture && showThumbnail"
-      :src="track.meta?.attachedPicture"
-      alt=""
-      class="rounded-md w-10 aspect-square bg-slate-100"
-    />
-    <div class="w-1/2">
-      <h4 class="font-semibold">{{ track.meta?.title }}</h4>
-      <span class="text-slate-700" v-if="track.meta?.artist">
-        {{ track.meta?.artist }}
-      </span>
-    </div>
-    <div class="">
-      <span class="text-slate-400">{{ track.subtype }}</span>
+  <li class="group relative py-3 cursor-pointer">
+    <div
+      class="opacity-0 group-hover:opacity-100 absolute -inset-y-2 -inset-x-4 rounded-xl bg-slate-100"
+    ></div>
+    <div class="relative flex gap-3 items-center justify-between">
+      <div
+        v-if="track.meta?.attachedPicture && showThumbnail"
+        class="relative w-10"
+      >
+        <ProtectedImage
+          :src="track.meta?.attachedPicture"
+          alt=""
+          class="rounded-md w-10 aspect-square bg-slate-100"
+        />
+        <div class="opacity-0 group-hover:opacity-100 absolute inset-0">
+          <div
+            class="absolute inset-0 rounded-md bg-black opacity-50 w-full h-full"
+          ></div>
+          <IconComponent
+            name="play"
+            filled
+            class="absolute inset-0 text-white text-2xl flex justify-center items-center"
+          />
+        </div>
+      </div>
+      <div class="mr-auto w-1/3">
+        <h4 class="font-semibold">{{ track.meta?.title }}</h4>
+        <span class="text-slate-700" v-if="track.meta?.artist">
+          {{ track.meta?.artist }}
+        </span>
+      </div>
+      <div
```

Comment:
Why not use `v-else` here? https://vuejs.org/guide/essentials/conditional.html#v-else

Reply:
they are quite far apart. I think it's more readable not to use it.